src/main/java/it/gov/pagopa/pu/bff/service/organization/BrokerServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package it.gov.pagopa.pu.bff.service.organization;
+
+import it.gov.pagopa.pu.bff.config.DefaultConfigFe;
+import it.gov.pagopa.pu.bff.connector.OrganizationClientImpl;
+import it.gov.pagopa.pu.bff.dto.generated.ConfigFE;
+import it.gov.pagopa.pu.bff.mapper.PersonalisationFE2ConfigFEMapper;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.EntityModelBroker;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.EntityModelOrganization;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.PersonalisationFe;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserInfo;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserOrganizationRoles;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BrokerServiceImpl implements BrokerService{
+
+  private OrganizationClientImpl organizationClient;
+
+  private final PersonalisationFE2ConfigFEMapper personalisationFE2ConfigFEMapper;
+
+  private DefaultConfigFe defaultConfigFe;
+
+  public BrokerServiceImpl(OrganizationClientImpl organizationClient,
+      DefaultConfigFe defaultConfigFe,
+      PersonalisationFE2ConfigFEMapper personalisationFE2ConfigFEMapper){
+    this.organizationClient = organizationClient;
+    this.defaultConfigFe = defaultConfigFe;
+    this.personalisationFE2ConfigFEMapper = personalisationFE2ConfigFEMapper;
+  }
+
+  @Override
+  public ConfigFE getBrokerConfig(UserInfo user, String accessToken){
+    UserOrganizationRoles orgRoles = user.getOrganizations().stream()
+      .filter(org -> StringUtils.isNotBlank(org.getOrganizationIpaCode()))
+      .findFirst()
+      .orElse(null);
```

Comment:
this way we will retrieve the right orgIpaCode to use to retrieve the Organization entity
```suggestion
    String orgIpaCode = Optional.ofNullable(user.getOrganizationAccess())
    .orElseGet(() -> user.getOrganizations().stream()
      .findFirst()
      .map(UserOrganizationRoles::getOrganizationIpaCode)
      .orElse(null)
      );
```